### PR TITLE
[Code] [Network] Network Player Camera

### PR DESCRIPTION
**Pull request by KristianKurk**
fixed up the camera for the network player
some of the player stuff doesn't work, but i'll focus on the lobby system first.

### CODE REVIEW

File: Project Bunny/Assets/Scripts/Player/NetworkStudentController.cs [hunk 1]
```diff
@@ -45,52 +57,106 @@ private void FixedUpdate()
...
+        public void SetCamera()
         {
-            _playerCamera = cam.transform.GetChild(0).GetComponent<Camera>();
-            _playerVirtualCamera = cam.GetComponent<CinemachineVirtualCamera>();
-            _playerVirtualCamera.Follow = gameObject.transform;
+            if (!_view.IsMine) return;
+
+            if (_playerCamera != null)
+            {
+                GameObject playerCamera = GameObject.Instantiate(_playerCameraPrefab, Vector3.zero, Quaternion.identity);
```
Comment (George540) [inline Project Bunny/Assets/Scripts/Player/NetworkStudentController.cs:78]: Put a TODO: Add pooling comment so I remember it when we decide to use pooling for instantiating dynanically

File: Project Bunny/Assets/Scripts/Player/NetworkStudentController.cs [hunk 2]
```diff
@@ -5,20 +5,31 @@
...
 namespace Player
 {
-    public class NetworkStudentController : MonoBehaviourPunCallbacks, IPunObservable
+    public class NetworkStudentController : MonoBehaviourPunCallbacks
     {
         [Header("Components")]
         [SerializeField] private Transform _playerModel;
-        [SerializeField] private Camera _playerCamera;
-        [SerializeField] private CinemachineVirtualCamera _playerVirtualCamera;
+        [SerializeField] private GameObject _playerCameraPrefab;
```
Comment (George540) [inline Project Bunny/Assets/Scripts/Player/NetworkStudentController.cs:12]: Why not put the camera Prefab in the NetworkManager.cs and spawn it there?

File: Project Bunny/Assets/Scripts/Player/NetworkStudentController.cs (hunk 1 above)
Comment (KristianKurk) [reply Project Bunny/Assets/Scripts/Player/NetworkStudentController.cs:78]: added

File: Project Bunny/Assets/Scripts/Player/NetworkStudentController.cs (hunk 2 above)
Comment (KristianKurk) [reply Project Bunny/Assets/Scripts/Player/NetworkStudentController.cs]: fixed

**Review by George540 — APPROVED** [review]
Good good. Can't wait to playtets the multiplayer part!

**Review by DrakeHunterK7 — APPROVED** [review]
Code looks good, and networking code seems like it should work. Once you address @George540 's comments above, you can go ahead and merge. Great work!

**Pull request merged**